static/styles/zulip.css

```diff
@@ -1148,13 +1148,25 @@ td.pointer {
     display: flex;
     align-items: center;
     justify-content: space-between;
+    padding: 8px 8px 8px 14px;
+    column-gap: 10px;
 
     #mark_as_read_turned_off_content {
         margin: 0;
+        flex-grow: 1;
     }
 
-    .mark_as_read_close {
-        margin-top: 6px;
+    #mark_as_read_controls {
+        display: flex;
+    }
+
+    #mark_as_read_close {
+        align-self: flex-start;
+        margin-top: -5px;
```

Comment:
What's the story with the `-5px` top margin? We generally try to avoid negative margins, since they often indicate some sort of bug elsewhere.

Reply:
<img width="637" alt="image" src="https://user-images.githubusercontent.com/5634097/176984048-24f43e59-6214-428e-9979-442c2b87ec83.png">

The error bar has shared margin/padding and the negative top margin gets the x button more into the corner. I think the option to avoid negative margin here is to put the text and orange button into a separate div so that they can share padding and not the x, but the way it currently is feels easier to wrap my head around so I'll leave it as that unless you have stronger preferences.

Reply:
OK, that makes sense. I think we're ultimately going to want to redo this type of widget as a proper component that we can use to replace bootstrap alert elements, and we can settle on a clean solution for this sort of detail when doing that.